resolwe/flow/managers/dispatcher.py

```diff
@@ -644,12 +656,7 @@ def _data_execute(self, data, program, executor, verbosity):
         if not program:
             return
 
-        logger.debug(__(
-            "Manager '{}.{}' preparing Data object {} for processing.",
-            self.__class__.__module__,
-            self.__class__.__name__,
-            data.id
-        ))
+        logger.debug(__("Manager preparing Data object {} for processing.", data.id))
```

Comment:
For consistency, I would change this to: `logger.debug(__("Manager preparing Data with id {} for processing.", data.id))`.